src/elements/Button/Button.tsx

```diff
@@ -101,17 +101,19 @@ export const Button: FC<ButtonProps & ButtonRef> = forwardRef(
         ref={ref}
         whileTap={{ scale: disabled || disableAnimation ? 1 : 0.9 }}
         data-variant={groupVariant || variant}
-        className={twMerge(
+        className={cn(
```

Comment:
Why change to `cn`?

Reply:
Because it union `twMerge` and `classNames` inside